demos/mediapipe/holistic_tracking/mediapipe_holistic_tracking.py

```diff
@@ -0,0 +1,197 @@
+    run_command("rm -rf saved_model")
+
+def download_model(model_path: str):
+    """Downloads the oss model from Google Cloud Storage if it doesn't exist in the package."""
+    model_url = _GCS_URL_PREFIX + model_path.split('/')[-1]
+    dst = os.path.join("ovms/", model_path.replace("/","/1/"))
+    dst_dir = os.path.dirname(model_path)
+
+    # Workaround to copy every model in separate directory
+    model_name = os.path.basename(model_path).replace(".tflite","")
+    dir_name = os.path.basename(dst_dir)
+    if dir_name != model_name:
+        dst = dst.replace(dir_name + "/", model_name + "/")
+
+    dst_dir = os.path.dirname(dst)
+    if not os.path.exists(dst_dir):
+        os.makedirs(dst_dir)
+
+    dst_file = os.path.join(dst_dir, os.path.basename(model_path))
+    print('Downloading model to ' + dst_file)
+    with urllib.request.urlopen(model_url) as response, open(dst_file,
+                                                           'wb') as out_file:
+        if response.code != 200:
+            raise ConnectionError('Cannot download ' + model_path +
+                                    ' from Google Cloud Storage.')
+        shutil.copyfileobj(response, out_file)
+
+def prepare_models():    
+    external_files = [
+       # Using short range
+       # 'face_detection/face_detection_full_range_sparse.tflite',
+        'face_detection/face_detection_short_range.tflite',
+        'face_landmark/face_landmark.tflite',
+       # Model loading error
+       # 'face_landmark/face_landmark_with_attention.tflite',
+        'hand_landmark/hand_landmark_full.tflite',
+       # Using full
+       # 'hand_landmark/hand_landmark_lite.tflite',
+        'holistic_landmark/hand_recrop.tflite',
+        'iris_landmark/iris_landmark.tflite',
+        'palm_detection/palm_detection_full.tflite',
+       # Using full
+       # 'palm_detection/palm_detection_lite.tflite',
+       # Need to use OV version
+        'pose_detection/pose_detection.tflite',
+        'pose_landmark/pose_landmark_full.tflite',
+       # Not working
+       # 'selfie_segmentation/selfie_segmentation.tflite',
+       # 'selfie_segmentation/selfie_segmentation_landscape.tflite',
+    ]
+    for elem in external_files:
+      sys.stderr.write('downloading file: %s\n' % elem)
+      download_model(elem)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Sends requests via KServe gRPC API using images in format supported by OpenCV. '
+                                                 'It displays performance statistics and optionally the model accuracy')
+    parser.add_argument('--images_list', required=False, default='input_images.txt', help='path to a file with a list of labeled images')
+    parser.add_argument('--grpc_address',required=False, default='localhost',  help='Specify url to grpc service. default:localhost')
+    parser.add_argument('--grpc_port',required=False, default=9022, help='Specify port to grpc service. default: 9022')

```

Comment:
i think that in every different demo we use default 9000

Reply:
Thats why i used 9022 in my demo :)